src/assets/captions/readme.md

```diff
@@ -79,17 +79,18 @@ No translations yet.
    	- Spanish; Kim Ortiz
    	- French; Béryl de La Grandière
 8. Let's Build an Accessible WordPress Page & Post
-9. How to Address Accessibility and Privacy to Help Avoid Fines and Lawsuits
    	- Spanish; Diana Córdoba Gómez
-10. Alt Text in the Age of Automation: How's it Actually Going?
+10. How to Address Accessibility and Privacy to Help Avoid Fines and Lawsuits
```

Comment:
Is there a reason the enumeration is changed here?

Reply:
Ugh. I think I know why. I have been updating this file in Github's web interface and I think it "helpfully" updated the numbers when I hit "enter" after the talk name to add a new line for the attribution.

I was tempted to use the markdown list numbering trick and just renumber all of them to 1, but I don't want to lose the numbering that we have with the multiple #15s (there's a 15th talk, plus all the lightning talks are labeled as #15). 

I hand-fixed the numbering, but this is going to be something to watch out for in the future.

Reply:
Hmmm. Yeah, that's a little bit weird with the numbering. Maybe we should just adjust to auto numbering; having all the lightning talks have the same number makes sense for the schedule, but I don't think it's really relevant here.